deps.edn

```diff
@@ -1,6 +1,7 @@
 {:deps {org.clojure/clojure                         {:mvn/version "1.10.3"}
         org.clojure/clojurescript                   {:mvn/version "1.11.4"}
         io.replikativ/hitchhiker-tree               {:mvn/version "0.1.11"}
+        io.replikativ/konserve                      {:mvn/version "0.6.0-alpha1"}
```

Comment:
I can see how that can fix the problem with different versions. Did you check if that is the case?

I only wonder if it would be better to place it above the hitchhiker-tree dependency because that should also pull in konserve

Reply:
Yes, I checked the versions selected for different builds.

As for dependency positioning, yes, hitchhiker-tree does pull in konserve. I compared the existing code and the change you suggested, but selected package versions were the same in both cases, so I'm keeping the status quo.

Reply:
Could you then go with the latest release of konserve `0.6.0-alpha3`. Or would that break something?